Formula/czmq.rb

```diff
@@ -45,9 +43,16 @@ def install
     system "make"
     system "(ZSYS_INTERFACE=lo0 && make check-verbose)"
     system "make", "install"
+    mv bin/"makecert", bin/"zmakecert"
     rm Dir["#{bin}/*.gsl"]
   end
 
+  def caveats; <<-EOS.undent
+      The 'makecert' binary has been installed as 'zmakecert' to avoid a conflict with the 'mono' formula.
+      This change will eventually be incorporated into a future CZMQ release. See https://github.com/zeromq/czmq/issues/1038 for more details.
```

Comment:
Can you wrap this message ensuring no more than 80 characters are output on a line? Thanks! I also think the second line can be a comment in the formula above and not printing as a caveat.